packages/core/src/__test__/selector.test.ts

```diff
@@ -0,0 +1,66 @@
+import { createSelector } from '../selector';
+import { memoize, shallowComparer } from '../utils';
+
+jest.mock('../utils');
+
+const memoizeMock = jest.mocked(memoize);
+memoizeMock.mockReturnValue(jest.fn);
+
+describe('core', () => {
```

Comment:
I suggest trying to rewrite this test suit: it should consider `createSelector` as a black box checking it's output. Instead it gets `createSelector` implementation and tests how functions calls each other

Reply:
I mean, the whole suit seems to be useless, like we don't have tests for `createSelector` at all

Reply:
I just moved the selector code and its tests from the existing ViewModel code.
So, this test isn't new and just was moved. (I added the last one only, which checks that we pass to memoize shallowComparer by default).
I'm confused about your comment because you developed these test cases.
[Link to your commit](https://github.com/DevExpress/DevExtreme/commit/74bef39f3398616b7f89357a44067f8c5a58c574#diff-b42bc75648481eb41f48c321dbdfdac6f7034d2e30c71c93fe0686174003457bR138)

I'm not against of change these test cases, but can you please provide an example of what you want?

Reply:
![image](https://user-images.githubusercontent.com/14272298/213409646-d4fb15e2-d0ab-4edb-8740-16a03ba39d1e.png)

Reply:
Okay then, if these are not new tests we should keep them as is.